src/lib/tip-claim-relay.js

```diff
@@ -1,6 +1,7 @@
 import { uniq } from 'lodash-es';
+import TIPPING_V1_INTERFACE from 'tipping-contract/Tipping_v1_Interface.aes';
```

Comment:
Shouldn't we also include the TIPPING_V2 contract here so we can pass claims for the new contract? 🤔